docker-compose.yml

```diff
@@ -93,6 +96,20 @@ services:
       timeout: 30s
       retries: 5
 
+  ipfs:
+    image: ipfs/kubo:master-2023-07-20-1fe17a4
+    container_name: indexer_ipfs
+    volumes:
+      - .ipfs/export:/export
```

Comment:
we have a `.data` for files, i think you can use `.data/ipfs` instead of `.ipfs`

Reply:
`.ipfs` replaced by `.data/ipfs` now